authors/apps/articles/views.py

```diff
@@ -44,6 +44,48 @@ def find_favorite(slug):
         })
 
 
+def get_highlights(slug):
+    """Method to get all highlights of an article by slug"""
+    return Highlight.objects.select_related(
+            'article').filter(article__slug=slug)
+
+
+def format_highlight(highlights, saved_article):
+    """Method to update start and end index of highlight if article
+    body is updated or deleted the highlight if idtdoes not exist anymore"""
```

Comment:
You have a minor typo here.

Reply:
@kirega Thanks for the feedback the typo has been corrected.